src/Curators/Serializer/NewsArticleNormalizer.php

```diff
@@ -9,15 +9,31 @@
 
 final class NewsArticleNormalizer implements NormalizerInterface
 {
+    private $jsonLd = false;
+
+    public function withJsonLd(bool $jsonLd): self
```

Comment:
I think we only want to turn jsonLd "on", so maybe we can call this `asJsonLd`?

Reply:
No real preference here but I'll change it!